tasks/lammps/run.py

```diff
@@ -114,14 +119,16 @@ def faasm(ctx, repeats=1, nprocs=None, procrange=None):
             start = time.time()
             stats.start_collection()
 
+            file_name = _bench["data"][0].split("/")[-1]
+            cmdline = "-in faasm://lammps-data/{}".format(file_name)
             url = "http://{}:{}".format(host, port)
             msg = {
                 "user": LAMMPS_FAASM_USER,
                 "function": LAMMPS_FAASM_FUNC,
-                "cmdline": LAMMPS_WASM_CMDLINE,
+                "cmdline": cmdline,
                 "mpi_world_size": int(np),
             }
-            print("Posting to {}".format(url))
+            print("Posting msg {} to {}".format(msg, url))
```

Comment:
This might not look particularly nice. You could instead use `pprint` to print the dictionary nicely on a new line, and put the URL before it.